src/Instrumentation/PDO/src/PDOInstrumentation.php

```diff
@@ -31,6 +31,56 @@ public static function register(): void
         );
         $pdoTracker = new PDOTracker();
 
+        // Hook for the new PDO::connect static method
+        if (method_exists(PDO::class, 'connect')) {
+            hook(
+                PDO::class,
+                'connect',
+                pre: static function (
+                    $object,
+                    array $params,
+                    string $class,
+                    string $function,
+                    ?string $filename,
+                    ?int $lineno,
+                ) use ($instrumentation) {
+                    /** @psalm-suppress ArgumentTypeCoercion */
+                    $builder = self::makeBuilder($instrumentation, 'PDO::connect', $function, $class, $filename, $lineno)
+                        ->setSpanKind(SpanKind::KIND_CLIENT);
+                    if ($class === PDO::class) {
+                        $builder
+                            ->setAttribute(TraceAttributes::SERVER_ADDRESS, $params[0] ?? 'unknown')
+                            ->setAttribute(TraceAttributes::SERVER_PORT, $params[0] ?? null);
+                    }
```

Comment:
The `if` condition is not required (only needed for `__construct` as constructors can be overridden with arbitrary parameters).

Should add the `//@todo` from `__construct()` / IMO we should not populate the `server.` attributes until the todo is fixed.